packages/auth/jest.config.json

```diff
@@ -4,5 +4,6 @@
   "testMatch": ["<rootDir>/src/__tests__/**/**.test.js"],
   "setupFilesAfterEnv": ["../../jestScaffolding.js", "./jest.setup.js"],
   "transformIgnorePatterns": ["/node_modules/"],
-  "testEnvironment": "node"
+  "testEnvironment": "node",
+  "testTimeout": 400000
```

Comment:
I believe so. I've seen once the `auth` failed in codeship in someone's test. Maybe we can make it lower. I've tried 50000 but it failed in my local so I give it one more '0'.

Reply:
Unit tests should run fairly quickly (if not, people will run them less often😛).

Do you remember which test case it was? I would be tempted to try making the tests quicker, rather than increasing the test timeout threshold. We can utilize mocking for this, or breaking the test into different cases